manifests/t/Tenable/Nessus/10.7.2/Tenable.Nessus.locale.en-US.yaml

```
@@ -0,0 +1,28 @@
+# Created with YamlCreate.ps1 v2.4.1 $debug=NVS1.CRLF.7-4-1.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Tenable.Nessus
+PackageVersion: 10.7.2
```

```suggestion
PackageVersion: 10.7.2.20029
```
Please update file name and path to match this change.